src/etc/inc/plugins.inc.d/unbound.inc

```diff
@@ -257,6 +257,7 @@ EOF;
 
     $forward_conf = '';
     $forward_local = '';
+    $resolv = '';
```

Comment:
last nit... I use grep like `grep resolv.conf.root` which can light up this variable too for easy code checks...

```suggestion
    $resolv_conf_root = '';
```